env

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+ORG_PATH="github.com/coreos"
+PROJ="mantle"
+REPO_PATH="${ORG_PATH}/${PROJ}"
+VERSION=$(git describe --dirty --always)
+GLDFLAGS="-X github.com/coreos/fleet/version.Version=\"${VERSION}\""
```

Comment:
fleet? delete please. If we want to start tagging and setting versions from tags that needs to be in a different PR